### DISCUSSION TITLE
Suggestions for features to implement next

### DISCUSSION THREAD

**Discussion by jkoutavas** [jkoutavas/Savitar2#33] [Ideas]
Savitar is in alpha now (see the [README](https://github.com/jkoutavas/Savitar2/blob/master/README.md)). Use this discussion topic to cover what you'd like to see implemented next.

**Comment by chriscampbell** [comment]
I do not have a machine to run the old version of Savitar on but I was able to get the alpha/beta running via Xcode. 

I'm up for throwing some time at this but not sure where I can contribute. 

There does some to be a minimal via product already available.

**Reply by jkoutavas** [reply]
@chriscampbell, how do you rate yourself on Swift development?

**Reply by jkoutavas** [reply]
Here's the train of thought that led things to the state of the app currently: [Savitar2DevNotes.md](https://github.com/jkoutavas/Savitar2/blob/master/docs/Savitar2DevNotes.md)
